RadixWallet/Clients/ProfileStore/ProfileStore.swift

```diff
@@ -61,6 +61,16 @@ extension ProfileStore {
 	}
 }
 
+extension ProfileStore {
+	public func isThisDevice(deviceID: DeviceID) async -> Bool {
```

Comment:
minor but I'd there is a possibility of "false negative" here, and thus might be more correct to call it `isKnownToBeThisDevice` or return `Bool?` and return `nil`  inside the guard. But yeah really minor...

Reply:
not your code...